var/spack/repos/builtin/packages/jq/package.py

```diff
@@ -34,3 +34,4 @@ class Jq(AutotoolsPackage):
     version('1.5', 'c8070bd6ec275404f77db3d2e568c9a3')
 
     depends_on('oniguruma')
+    depends_on('bison@3.0:')
```

Comment:
ps. I'm a bit confused why I'm stumbling on this now.  I've changed a couple of things trying to get my Lmod modules working well pulled in a few more commits and am probably suffering from changing several things at a time....